docs/mathematics/happy-number.mdx

```diff
@@ -0,0 +1,244 @@
+---
+title: Happy Number
+slug: /mathematics/happy-number
+---
+
+## Introduction
+
+A number is called a happy number if it converges to 1 after a sequential operation wherein each step we replace the original number by the sum of the squares of its individual digits.When we have a digit given to us, we perform the operation, the sum of squares of its individual digits but in some cases, we might arrive at the same number which we already came across. In such cases, we can clearly see that it would never terminate so such numbers are not considered as happy numbers.
+
+## Explanation
+
+If the number we calculate by the sum of squares appears again in the loop. In such a case, we don't see the loop terminating. So such a number is not a happy number.
```

Comment:
Replace this with: 

`If the number we are processing by calculating their sum of squares appears again in the loop, we will not see the loop terminating. Such a number cannot be considered as a happy number.`